integration_tests/test_reward.py

```diff
@@ -19,7 +19,9 @@ def test_reward(cluster):
     fees = 3_000_000_000
     # wait for initial reward processed, so that distribution values can be read
     wait_for_block(cluster, 2)
-    old_commission_amount = cluster.distribution_commission(validator1_operator_address)
+    old_commission_amount = (
+        0  # cluster.distribution_commission(validator1_operator_address)
```

Comment:
this comment cause a linter issue. BTW, why comment out the call?

Reply:
because the config changed to have 0 commission -- that call failed due to that: https://github.com/crypto-org-chain/chain-main/runs/7912468478?check_suite_focus=true

I could possibly prepare a separate config for upgrade, but it seems simpler to just adjust this test